roles/cinder-common/templates/etc/cinder/cinder.conf

```diff
@@ -36,46 +36,53 @@ allow_availability_zone_fallback = {{ cinder.allow_availability_zone_fallback }}
 volume_clear_size = {{ cinder.volume_clear_size }}
 {% endif -%}
 
-{% if ceph.enabled|bool -%}
-{% if cinder.common_volume_name -%}
-host = ceph
-{% endif -%}
-{% endif -%}
-
 glance_host = {{ endpoints.main }}
 glance_port = {{ endpoints.glance.port.backend_api }}
 
-{% if cinder.enabled_backends|length -%}
-enabled_backends = {{ cinder.enabled_backends|join(',') }}
-{% endif %}
-{% if cinder.default_backend != None -%}
-default_backend = {{ cinder.default_backend }}
-{% endif %}
-
 {% if swift.enabled|bool -%}
 backup_driver = cinder.backup.drivers.swift
 backup_swift_url = http://{{ endpoints.swift.haproxy_vip }}:{{ endpoints.swift.port.cinder_backup }}/v1/
 {% endif -%}
 
-{% for backend in cinder.backends %}
-[{{ backend.name }}]
-volume_backend_name = {{ backend.name }}
-volume_driver = {{ backend.volume_driver }}
-{% if backend.name == "rbd_volumes" %}
-rbd_pool = {{ ceph.pool_name }}
+{% if ceph.enabled|bool -%}
+{% if cinder.common_volume_name -%}
+host = ceph
+{% endif -%}
+
+{% if cinder.default_volume_type | default(false) -%}
+default_volume_type = {{ cinder.default_volume_type }}
+{% else %}
+default_volume_type = {{ ceph_pools[ceph_default_backend]['volume_type'] }}
```

Comment:
This might fail when one runs with --tags cinder  as ceph_pools is defined in ceph-osd role.